taca/storage/storage.py

```diff
@@ -229,11 +229,11 @@ def _send_to_swestore(f, dest, remove=True):
             misc.call_external_command('iput -K -P {file} {dest}'.format(file=f, dest=dest),
                     with_log_files=True)
             logger.info('Run {} sent correctly and checksum was okay.'.format(f))
+            if remove:
+                logger.info('Removing run'.format(f))
+                os.remove(f)
```

Comment:
no that's the weird part, I'll have to check, because this is running in the NASs and I would swear that data was being deleted...